packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -5,10 +5,11 @@ const { CryptoWatchPriceFeed } = require("./CryptoWatchPriceFeed");
 const { UniswapPriceFeed } = require("./UniswapPriceFeed");
 const { BalancerPriceFeed } = require("./BalancerPriceFeed");
 const { defaultConfigs } = require("./DefaultPriceFeedConfigs");
+const { getTruffleContract } = require("@uma/core");
 
-const Uniswap = require("@uma/core/build/contracts/Uniswap.json");
-const ExpiringMultiParty = require("@uma/core/build/contracts/ExpiringMultiParty.json");
-const Balancer = require("@uma/core/build/contracts/Balancer.json");
+const ExpiringMultiParty = getTruffleContract("ExpiringMultiParty", "1.2.0");
```

Comment:
The second argument to `getTruffleContract` is web3 (you can also set it to null/undefined and it will try to grab it from global.web3).

I think that's why it's not respecting your versions strings.

Reply:
ahhhh let me try that!